src/viewer.js

```diff
@@ -328,6 +328,22 @@ $.Viewer = function( options ) {
 
     THIS[ this.hash ].prevContainerSize = _getSafeElemSize( this.container );
 
+    this._onViewerResize = onViewerResize;
+    this._origViewerResize = origViewerResize; //for testing logic changes
+    if(ResizeObserver){
```

Comment:
I imagine this probably should be `window.ResizeObserver`. Otherwise, if it's missing, it'll just throw an exception.